bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: argocd-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.10.0+git
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0
```

Comment:
@raghavi101 I don't think we want to make this change here

Reply:
Yeah I ran `make bundle` locally to fix the failing pipeline. Trying to figure out how to fix that CI fail. Will revert these changes back